includes/application.php

```diff
@@ -0,0 +1,12 @@
+<?php
+	$APPLICATION_DIR = str_replace("includes","", dirname(__FILE__));
+
+	spl_autoload_register(function ($class_name) {
+		global $APPLICATION_DIR;
+	    include $APPLICATION_DIR.'/modules/'.$class_name . '.php';
+	});
+	
+	require_once($APPLICATION_DIR.'/includes/settings.php');
```

Comment:
Add this in the `composer autoloader`
Check PSR-0 or PSR-4 file inclusion using autoloader for this.